charts/victoria-metrics-agent/templates/podsecuritypolicy.yaml

```diff
@@ -3,17 +3,14 @@ apiVersion: policy/v1beta1
 kind: PodSecurityPolicy
```

Comment:
I think we can remove PodSecurityPolicy from our charts totally. It was removed in k8s v1.25, and we claimed to support >v1.27 recently.
wdyt?

Reply:
removed